javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/RemovedGroup.java

```diff
@@ -0,0 +1,215 @@
+package com.github.javaparser.printer.lexicalpreservation;
+
+import com.github.javaparser.JavaToken;
+import com.github.javaparser.TokenRange;
+import com.github.javaparser.TokenTypes;
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.printer.concretesyntaxmodel.CsmToken;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+class RemovedGroup implements Iterable<Removed> {
```

Comment:
What this class represent?

Reply:
Thanks, I will add a JavaDoc when I get home.